app/src/js/components/Errors/copy-to-clipboard.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const CopyToClipboard = ({
+  text,
+}) => {
+  const [copy, copiedTextState] = useState('Copy');
+
+  async function handleClick(e) {
+    e.preventDefault();
+    try {
+      await navigator.clipboard.writeText(text);
+      copiedTextState('Copied!');
+    } catch (err) {
+      copiedTextState('Not Copied!');
```

Comment:
Nit: can we make this more descriptive with something like 'Error copying'. I don't know that we need to display the error itself, but I think it would make it more clear that something happened while trying to copy.